### PR TITLE
Dev/verification backend

### PR DESCRIPTION
**Pull request by hochangyong**
### **Done User Verification Backend**

_** Thanks for the help from jackson and junyao_

### CODE REVIEW

File: public/js/userVerification.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+for (let acceptBtn of acceptBtns) {
+    acceptBtn.addEventListener('click', function () {
+        console.log(acceptBtn.parentElement.dataset.userId);
+        updateVerification(acceptBtn.parentElement.dataset.userId);
+        verifContainer.style.display = "none";
+    })
+}
+
+let rejectBtns = document.querySelectorAll('.user-verification-btn--reject');
+let rejectBtnContainer = document.querySelector('.user-verification-wrapper');
```
Comment (JacksonTai) [inline public/js/userVerification.js:13]: What does line 13 for?

Comment (hochangyong) [reply public/js/userVerification.js:13]: oh sorry, forgot to remove it, let me commit and push again

**Review by JacksonTai — DISMISSED** [review]
You forgot to insert the NRIC no to user_detail table upon accepting the verification request.

**Comment by hochangyong** [comment]
thank you so much jackson, so far everything work and didn't saw any problem

**Review by JacksonTai — APPROVED** [review]

**Review by EHeng17 — APPROVED** [review]

**Review by jyyyyylim — APPROVED** [review]

**Comment by jyyyyylim** [comment]
flawless, thanks for reminding me

**Pull request merged**